public/js/lockToggle.js

```diff
@@ -0,0 +1,7 @@
+
+ const toggleLocks = document.querySelectorAll('#lockToggle')
+     toggleLocks.forEach(toggleLock => {
+         toggleLock.addEventListener('click',(e) => {
+             e.target.classList.toggle("bi-lock");
+             e.target.classList.toggle("bi-unlock");
+         })});
```

Comment:
i think the second set of }); needs to be on line 8 in line with the toggleLocks.forEach on line 3